? wx-xrc-classinfo.patch
? include/wx/stc/stc.patch
? samples/xrc/xrcdemo.ilk
? src/fl/Debug_DLL
? src/fl/flVC___Win32_Debug_Unicode_DLL
? src/stc/Debug_DLL
? src/stc/Release_DLL
Index: include/wx/xrc/xh_bmp.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_bmp.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_bmp.h
--- include/wx/xrc/xh_bmp.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_bmp.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxBitmapXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxBitmapXmlHandler)
 public:
     wxBitmapXmlHandler();
@@ -29,4 +30,5 @@
 class WXXMLDLLEXPORT wxIconXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxIconXmlHandler)
 public:
     wxIconXmlHandler();
Index: include/wx/xrc/xh_bmpbt.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_bmpbt.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_bmpbt.h
--- include/wx/xrc/xh_bmpbt.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_bmpbt.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxBitmapButtonXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxBitmapButtonXmlHandler)
 public:
     wxBitmapButtonXmlHandler();
Index: include/wx/xrc/xh_bttn.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_bttn.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_bttn.h
--- include/wx/xrc/xh_bttn.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_bttn.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxButtonXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxButtonXmlHandler)
 public:
     wxButtonXmlHandler();
Index: include/wx/xrc/xh_cald.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_cald.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_cald.h
--- include/wx/xrc/xh_cald.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_cald.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxCalendarCtrlXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxCalendarCtrlXmlHandler)
 public:
     wxCalendarCtrlXmlHandler();
Index: include/wx/xrc/xh_chckb.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_chckb.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_chckb.h
--- include/wx/xrc/xh_chckb.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_chckb.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxCheckBoxXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxCheckBoxXmlHandler)
 public:
     wxCheckBoxXmlHandler();
Index: include/wx/xrc/xh_chckl.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_chckl.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_chckl.h
--- include/wx/xrc/xh_chckl.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_chckl.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxCheckListXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxCheckListXmlHandler)
 public:
     wxCheckListXmlHandler();
Index: include/wx/xrc/xh_choic.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_choic.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_choic.h
--- include/wx/xrc/xh_choic.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_choic.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxChoiceXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxChoiceXmlHandler)
 public:
     wxChoiceXmlHandler();
Index: include/wx/xrc/xh_combo.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_combo.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_combo.h
--- include/wx/xrc/xh_combo.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_combo.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxComboBoxXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxComboBoxXmlHandler)
 public:
     wxComboBoxXmlHandler();
Index: include/wx/xrc/xh_dlg.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_dlg.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_dlg.h
--- include/wx/xrc/xh_dlg.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_dlg.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxDialogXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxDialogXmlHandler)
 public:
     wxDialogXmlHandler();
Index: include/wx/xrc/xh_frame.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_frame.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_frame.h
--- include/wx/xrc/xh_frame.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_frame.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxFrameXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxFrameXmlHandler)
 public:
     wxFrameXmlHandler();
Index: include/wx/xrc/xh_gauge.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_gauge.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_gauge.h
--- include/wx/xrc/xh_gauge.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_gauge.h	2003/06/12 00:03:50
@@ -25,4 +25,5 @@
 class WXXMLDLLEXPORT wxGaugeXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxGaugeXmlHandler)
     enum
     {
Index: include/wx/xrc/xh_gdctl.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_gdctl.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_gdctl.h
--- include/wx/xrc/xh_gdctl.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_gdctl.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxGenericDirCtrlXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxGenericDirCtrlXmlHandler)
 public:
     wxGenericDirCtrlXmlHandler();
Index: include/wx/xrc/xh_html.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_html.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_html.h
--- include/wx/xrc/xh_html.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_html.h	2003/06/12 00:03:50
@@ -24,4 +24,5 @@
 class WXXMLDLLEXPORT wxHtmlWindowXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxHtmlWindowXmlHandler)
 public:
     wxHtmlWindowXmlHandler();
Index: include/wx/xrc/xh_listb.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_listb.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_listb.h
--- include/wx/xrc/xh_listb.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_listb.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxListBoxXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxListBoxXmlHandler)
 public:
     wxListBoxXmlHandler();
Index: include/wx/xrc/xh_listc.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_listc.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_listc.h
--- include/wx/xrc/xh_listc.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_listc.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxListCtrlXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxListCtrlXmlHandler)
 public:
     wxListCtrlXmlHandler();
Index: include/wx/xrc/xh_menu.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_menu.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_menu.h
--- include/wx/xrc/xh_menu.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_menu.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxMenuXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxMenuXmlHandler)
 public:
     wxMenuXmlHandler();
@@ -31,4 +32,5 @@
 class WXXMLDLLEXPORT wxMenuBarXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxMenuBarXmlHandler)
     public:
         wxMenuBarXmlHandler();
Index: include/wx/xrc/xh_notbk.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_notbk.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_notbk.h
--- include/wx/xrc/xh_notbk.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_notbk.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxNotebookXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxNotebookXmlHandler)
 public:
     wxNotebookXmlHandler();
Index: include/wx/xrc/xh_panel.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_panel.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_panel.h
--- include/wx/xrc/xh_panel.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_panel.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxPanelXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxPanelXmlHandler)
 public:
     wxPanelXmlHandler();
Index: include/wx/xrc/xh_radbt.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_radbt.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_radbt.h
--- include/wx/xrc/xh_radbt.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_radbt.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxRadioButtonXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxRadioButtonXmlHandler)
 public:
     wxRadioButtonXmlHandler();
Index: include/wx/xrc/xh_radbx.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_radbx.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_radbx.h
--- include/wx/xrc/xh_radbx.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_radbx.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxRadioBoxXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxRadioBoxXmlHandler)
 public:
     wxRadioBoxXmlHandler();
Index: include/wx/xrc/xh_scrol.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_scrol.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_scrol.h
--- include/wx/xrc/xh_scrol.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_scrol.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxScrollBarXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxScrollBarXmlHandler)
     enum
     {
Index: include/wx/xrc/xh_scwin.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_scwin.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_scwin.h
--- include/wx/xrc/xh_scwin.h	2002/12/04 13:53:04	1.2
+++ include/wx/xrc/xh_scwin.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxScrolledWindowXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxScrolledWindowXmlHandler)
 public:
     wxScrolledWindowXmlHandler();
Index: include/wx/xrc/xh_sizer.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_sizer.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_sizer.h
--- include/wx/xrc/xh_sizer.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_sizer.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxSizerXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxSizerXmlHandler)
 public:
     wxSizerXmlHandler();
Index: include/wx/xrc/xh_slidr.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_slidr.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_slidr.h
--- include/wx/xrc/xh_slidr.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_slidr.h	2003/06/12 00:03:50
@@ -23,4 +23,5 @@
 class WXXMLDLLEXPORT wxSliderXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxSliderXmlHandler)
     enum
     {
Index: include/wx/xrc/xh_spin.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_spin.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_spin.h
--- include/wx/xrc/xh_spin.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_spin.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxSpinButtonXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxSpinButtonXmlHandler)
     enum
     {
@@ -39,4 +40,5 @@
 class WXXMLDLLEXPORT wxSpinCtrlXmlHandler : public wxXmlResourceHandler
 {
+	DECLARE_DYNAMIC_CLASS(wxSpinCtrlXmlHandler)
     enum
     {
Index: include/wx/xrc/xh_split.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_split.h,v
retrieving revision 1.1
diff -b -u -2 -r1.1 xh_split.h
--- include/wx/xrc/xh_split.h	2003/01/26 23:17:34	1.1
+++ include/wx/xrc/xh_split.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxSplitterWindowXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxSplitterWindowXmlHandler)
 public:
     wxSplitterWindowXmlHandler();
Index: include/wx/xrc/xh_stbmp.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_stbmp.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_stbmp.h
--- include/wx/xrc/xh_stbmp.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_stbmp.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxStaticBitmapXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxStaticBitmapXmlHandler)
 public:
     wxStaticBitmapXmlHandler();
Index: include/wx/xrc/xh_stbox.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_stbox.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_stbox.h
--- include/wx/xrc/xh_stbox.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_stbox.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxStaticBoxXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxStaticBoxXmlHandler)
 public:
     wxStaticBoxXmlHandler();
Index: include/wx/xrc/xh_stlin.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_stlin.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_stlin.h
--- include/wx/xrc/xh_stlin.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_stlin.h	2003/06/12 00:03:50
@@ -22,4 +22,5 @@
 class WXXMLDLLEXPORT wxStaticLineXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxStaticLineXmlHandler)
 public:
     wxStaticLineXmlHandler();
Index: include/wx/xrc/xh_sttxt.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_sttxt.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_sttxt.h
--- include/wx/xrc/xh_sttxt.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_sttxt.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxStaticTextXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxStaticTextXmlHandler)
 public:
     wxStaticTextXmlHandler();
Index: include/wx/xrc/xh_text.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_text.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_text.h
--- include/wx/xrc/xh_text.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_text.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxTextCtrlXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxTextCtrlXmlHandler)
 public:
     wxTextCtrlXmlHandler();
Index: include/wx/xrc/xh_toolb.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_toolb.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_toolb.h
--- include/wx/xrc/xh_toolb.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_toolb.h	2003/06/12 00:03:50
@@ -24,4 +24,5 @@
 class WXXMLDLLEXPORT wxToolBarXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxToolBarXmlHandler)
 public:
     wxToolBarXmlHandler();
Index: include/wx/xrc/xh_tree.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_tree.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_tree.h
--- include/wx/xrc/xh_tree.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_tree.h	2003/06/12 00:03:50
@@ -20,4 +20,5 @@
 class WXXMLDLLEXPORT wxTreeCtrlXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxTreeCtrlXmlHandler)
 public:
     wxTreeCtrlXmlHandler();
Index: include/wx/xrc/xh_unkwn.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_unkwn.h,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_unkwn.h
--- include/wx/xrc/xh_unkwn.h	2002/09/07 12:10:21	1.2
+++ include/wx/xrc/xh_unkwn.h	2003/06/12 00:03:50
@@ -21,4 +21,5 @@
 class WXXMLDLLEXPORT wxUnknownWidgetXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxUnknownWidgetXmlHandler)
 public:
     wxUnknownWidgetXmlHandler();
Index: include/wx/xrc/xh_wizrd.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xh_wizrd.h,v
retrieving revision 1.1
diff -b -u -2 -r1.1 xh_wizrd.h
--- include/wx/xrc/xh_wizrd.h	2003/04/22 20:25:24	1.1
+++ include/wx/xrc/xh_wizrd.h	2003/06/12 00:03:50
@@ -24,4 +24,5 @@
 class WXXMLDLLEXPORT wxWizardXmlHandler : public wxXmlResourceHandler
 {
+DECLARE_DYNAMIC_CLASS(wxWizardXmlHandler)
 public:
     wxWizardXmlHandler();
Index: include/wx/xrc/xmlres.h
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/include/wx/xrc/xmlres.h,v
retrieving revision 1.23
diff -b -u -2 -r1.23 xmlres.h
--- include/wx/xrc/xmlres.h	2003/04/07 22:15:03	1.23
+++ include/wx/xrc/xmlres.h	2003/06/12 00:03:51
@@ -303,4 +303,5 @@
 class WXXMLDLLEXPORT wxXmlResourceHandler : public wxObject
 {
+DECLARE_ABSTRACT_CLASS(wxXmlResourceHandler)
 public:
     // Constructor.
Index: src/xrc/xh_bmp.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_bmp.cpp,v
retrieving revision 1.3
diff -b -u -2 -r1.3 xh_bmp.cpp
--- src/xrc/xh_bmp.cpp	2001/12/29 16:14:04	1.3
+++ src/xrc/xh_bmp.cpp	2003/06/12 00:03:51
@@ -23,4 +23,5 @@
 #include "wx/bitmap.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxBitmapXmlHandler, wxXmlResourceHandler)
 
 wxBitmapXmlHandler::wxBitmapXmlHandler() 
@@ -38,4 +39,6 @@
     return IsOfClass(node, wxT("wxBitmap"));
 }
+
+IMPLEMENT_DYNAMIC_CLASS(wxIconXmlHandler, wxXmlResourceHandler)
 
 wxIconXmlHandler::wxIconXmlHandler() 
Index: src/xrc/xh_bmpbt.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_bmpbt.cpp,v
retrieving revision 1.3
diff -b -u -2 -r1.3 xh_bmpbt.cpp
--- src/xrc/xh_bmpbt.cpp	2001/12/29 16:14:04	1.3
+++ src/xrc/xh_bmpbt.cpp	2003/06/12 00:03:51
@@ -23,4 +23,6 @@
 #include <wx/bmpbuttn.h>
 
+IMPLEMENT_DYNAMIC_CLASS(wxBitmapButtonXmlHandler, wxXmlResourceHandler)
+
 wxBitmapButtonXmlHandler::wxBitmapButtonXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_bttn.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_bttn.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_bttn.cpp
--- src/xrc/xh_bttn.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_bttn.cpp	2003/06/12 00:03:51
@@ -23,4 +23,5 @@
 #include "wx/button.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxButtonXmlHandler, wxXmlResourceHandler)
 
 wxButtonXmlHandler::wxButtonXmlHandler() 
Index: src/xrc/xh_cald.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_cald.cpp,v
retrieving revision 1.5
diff -b -u -2 -r1.5 xh_cald.cpp
--- src/xrc/xh_cald.cpp	2002/04/09 19:34:59	1.5
+++ src/xrc/xh_cald.cpp	2003/06/12 00:03:51
@@ -28,4 +28,5 @@
 #include "wx/calctrl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxCalendarCtrlXmlHandler, wxXmlResourceHandler)
 
 wxCalendarCtrlXmlHandler::wxCalendarCtrlXmlHandler() 
Index: src/xrc/xh_chckb.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_chckb.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_chckb.cpp
--- src/xrc/xh_chckb.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_chckb.cpp	2003/06/12 00:03:51
@@ -25,4 +25,6 @@
 #if wxUSE_CHECKBOX
 
+IMPLEMENT_DYNAMIC_CLASS(wxCheckBoxXmlHandler, wxXmlResourceHandler)
+
 wxCheckBoxXmlHandler::wxCheckBoxXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_chckl.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_chckl.cpp,v
retrieving revision 1.10
diff -b -u -2 -r1.10 xh_chckl.cpp
--- src/xrc/xh_chckl.cpp	2003/05/04 14:11:30	1.10
+++ src/xrc/xh_chckl.cpp	2003/06/12 00:03:51
@@ -26,4 +26,6 @@
 #include "wx/intl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxCheckListXmlHandler, wxXmlResourceHandler)
+
 wxCheckListXmlHandler::wxCheckListXmlHandler() 
 : wxXmlResourceHandler(), m_insideBox(FALSE)
Index: src/xrc/xh_choic.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_choic.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_choic.cpp
--- src/xrc/xh_choic.cpp	2002/09/01 17:11:38	1.6
+++ src/xrc/xh_choic.cpp	2003/06/12 00:03:51
@@ -24,4 +24,6 @@
 #include "wx/intl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxChoiceXmlHandler, wxXmlResourceHandler)
+
 wxChoiceXmlHandler::wxChoiceXmlHandler() 
 : wxXmlResourceHandler() , m_insideBox(FALSE)
Index: src/xrc/xh_combo.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_combo.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_combo.cpp
--- src/xrc/xh_combo.cpp	2002/09/01 17:11:38	1.6
+++ src/xrc/xh_combo.cpp	2003/06/12 00:03:51
@@ -26,4 +26,6 @@
 #if wxUSE_COMBOBOX
 
+IMPLEMENT_DYNAMIC_CLASS(wxComboBoxXmlHandler, wxXmlResourceHandler)
+
 wxComboBoxXmlHandler::wxComboBoxXmlHandler() 
 : wxXmlResourceHandler() , m_insideBox(FALSE)
Index: src/xrc/xh_dlg.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_dlg.cpp,v
retrieving revision 1.9
diff -b -u -2 -r1.9 xh_dlg.cpp
--- src/xrc/xh_dlg.cpp	2003/02/28 22:09:14	1.9
+++ src/xrc/xh_dlg.cpp	2003/06/12 00:03:51
@@ -25,4 +25,5 @@
 #include "wx/intl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxDialogXmlHandler, wxXmlResourceHandler)
 
 wxDialogXmlHandler::wxDialogXmlHandler() : wxXmlResourceHandler()
Index: src/xrc/xh_frame.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_frame.cpp,v
retrieving revision 1.7
diff -b -u -2 -r1.7 xh_frame.cpp
--- src/xrc/xh_frame.cpp	2003/02/28 22:09:14	1.7
+++ src/xrc/xh_frame.cpp	2003/06/12 00:03:51
@@ -25,4 +25,5 @@
 #include "wx/intl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxFrameXmlHandler, wxXmlResourceHandler)
 
 wxFrameXmlHandler::wxFrameXmlHandler() : wxXmlResourceHandler()
Index: src/xrc/xh_gauge.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_gauge.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_gauge.cpp
--- src/xrc/xh_gauge.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_gauge.cpp	2003/06/12 00:03:51
@@ -25,4 +25,6 @@
 #if wxUSE_GAUGE
 
+IMPLEMENT_DYNAMIC_CLASS(wxGaugeXmlHandler, wxXmlResourceHandler)
+
 wxGaugeXmlHandler::wxGaugeXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_gdctl.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_gdctl.cpp,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_gdctl.cpp
--- src/xrc/xh_gdctl.cpp	2003/04/10 23:00:58	1.2
+++ src/xrc/xh_gdctl.cpp	2003/06/12 00:03:51
@@ -27,4 +27,6 @@
 #include "wx/dirctrl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxGenericDirCtrlXmlHandler, wxXmlResourceHandler)
+
 wxGenericDirCtrlXmlHandler::wxGenericDirCtrlXmlHandler()
 : wxXmlResourceHandler()
Index: src/xrc/xh_html.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_html.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_html.cpp
--- src/xrc/xh_html.cpp	2003/06/05 21:33:56	1.6
+++ src/xrc/xh_html.cpp	2003/06/12 00:03:51
@@ -27,4 +27,5 @@
 #include "wx/filesys.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxHtmlWindowXmlHandler, wxXmlResourceHandler)
 
 wxHtmlWindowXmlHandler::wxHtmlWindowXmlHandler() 
Index: src/xrc/xh_listb.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_listb.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_listb.cpp
--- src/xrc/xh_listb.cpp	2002/09/01 17:11:38	1.6
+++ src/xrc/xh_listb.cpp	2003/06/12 00:03:51
@@ -24,4 +24,6 @@
 #include "wx/intl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxListBoxXmlHandler, wxXmlResourceHandler)
+
 wxListBoxXmlHandler::wxListBoxXmlHandler() 
 : wxXmlResourceHandler() , m_insideBox(FALSE)
Index: src/xrc/xh_listc.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_listc.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_listc.cpp
--- src/xrc/xh_listc.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_listc.cpp	2003/06/12 00:03:51
@@ -25,4 +25,6 @@
 
 
+IMPLEMENT_DYNAMIC_CLASS(wxListCtrlXmlHandler, wxXmlResourceHandler)
+
 wxListCtrlXmlHandler::wxListCtrlXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_menu.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_menu.cpp,v
retrieving revision 1.10
diff -b -u -2 -r1.10 xh_menu.cpp
--- src/xrc/xh_menu.cpp	2003/05/04 14:11:30	1.10
+++ src/xrc/xh_menu.cpp	2003/06/12 00:03:51
@@ -24,4 +24,5 @@
 #include "wx/frame.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxMenuXmlHandler, wxXmlResourceHandler)
 
 wxMenuXmlHandler::wxMenuXmlHandler() : 
@@ -110,4 +111,6 @@
            );
 }
+
+IMPLEMENT_DYNAMIC_CLASS(wxMenuBarXmlHandler, wxXmlResourceHandler)
 
 wxMenuBarXmlHandler::wxMenuBarXmlHandler() : wxXmlResourceHandler()
Index: src/xrc/xh_notbk.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_notbk.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_notbk.cpp
--- src/xrc/xh_notbk.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_notbk.cpp	2003/06/12 00:03:51
@@ -28,4 +28,6 @@
 #include "wx/sizer.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxNotebookXmlHandler, wxXmlResourceHandler)
+
 wxNotebookXmlHandler::wxNotebookXmlHandler() 
 : wxXmlResourceHandler(), m_isInside(FALSE), m_notebook(NULL)
Index: src/xrc/xh_panel.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_panel.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_panel.cpp
--- src/xrc/xh_panel.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_panel.cpp	2003/06/12 00:03:51
@@ -23,4 +23,5 @@
 #include "wx/panel.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxPanelXmlHandler, wxXmlResourceHandler)
 
 wxPanelXmlHandler::wxPanelXmlHandler() : wxXmlResourceHandler()
Index: src/xrc/xh_radbt.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_radbt.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_radbt.cpp
--- src/xrc/xh_radbt.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_radbt.cpp	2003/06/12 00:03:51
@@ -25,4 +25,6 @@
 #if wxUSE_RADIOBOX
 
+IMPLEMENT_DYNAMIC_CLASS(wxRadioButtonXmlHandler, wxXmlResourceHandler)
+
 wxRadioButtonXmlHandler::wxRadioButtonXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_radbx.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_radbx.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_radbx.cpp
--- src/xrc/xh_radbx.cpp	2002/09/01 17:11:38	1.6
+++ src/xrc/xh_radbx.cpp	2003/06/12 00:03:51
@@ -26,4 +26,6 @@
 #if wxUSE_RADIOBOX
 
+IMPLEMENT_DYNAMIC_CLASS(wxRadioBoxXmlHandler, wxXmlResourceHandler)
+
 wxRadioBoxXmlHandler::wxRadioBoxXmlHandler() 
 : wxXmlResourceHandler(), m_insideBox(FALSE)
Index: src/xrc/xh_scrol.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_scrol.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_scrol.cpp
--- src/xrc/xh_scrol.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_scrol.cpp	2003/06/12 00:03:51
@@ -23,4 +23,5 @@
 #include "wx/scrolbar.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxScrollBarXmlHandler, wxXmlResourceHandler)
 
 wxScrollBarXmlHandler::wxScrollBarXmlHandler() 
Index: src/xrc/xh_scwin.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_scwin.cpp,v
retrieving revision 1.2
diff -b -u -2 -r1.2 xh_scwin.cpp
--- src/xrc/xh_scwin.cpp	2002/12/04 13:55:29	1.2
+++ src/xrc/xh_scwin.cpp	2003/06/12 00:03:51
@@ -23,4 +23,5 @@
 #include "wx/scrolwin.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxScrolledWindowXmlHandler, wxXmlResourceHandler)
 
 wxScrolledWindowXmlHandler::wxScrolledWindowXmlHandler() 
Index: src/xrc/xh_sizer.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_sizer.cpp,v
retrieving revision 1.8
diff -b -u -2 -r1.8 xh_sizer.cpp
--- src/xrc/xh_sizer.cpp	2003/04/22 20:26:06	1.8
+++ src/xrc/xh_sizer.cpp	2003/06/12 00:03:52
@@ -28,4 +28,6 @@
 #include "wx/tokenzr.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxSizerXmlHandler, wxXmlResourceHandler)
+
 bool wxSizerXmlHandler::IsSizerNode(wxXmlNode *node)
 {
Index: src/xrc/xh_slidr.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_slidr.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_slidr.cpp
--- src/xrc/xh_slidr.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_slidr.cpp	2003/06/12 00:03:52
@@ -25,4 +25,6 @@
 #if wxUSE_SLIDER
 
+IMPLEMENT_DYNAMIC_CLASS(wxSliderXmlHandler, wxXmlResourceHandler)
+
 wxSliderXmlHandler::wxSliderXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_spin.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_spin.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_spin.cpp
--- src/xrc/xh_spin.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_spin.cpp	2003/06/12 00:03:52
@@ -25,4 +25,6 @@
 #if wxUSE_SPINBTN
 
+IMPLEMENT_DYNAMIC_CLASS(wxSpinButtonXmlHandler, wxXmlResourceHandler)
+
 wxSpinButtonXmlHandler::wxSpinButtonXmlHandler() 
 : wxXmlResourceHandler() 
@@ -61,4 +63,6 @@
 
 #if wxUSE_SPINCTRL
+
+IMPLEMENT_DYNAMIC_CLASS(wxSpinCtrlXmlHandler, wxXmlResourceHandler)
 
 wxSpinCtrlXmlHandler::wxSpinCtrlXmlHandler() 
Index: src/xrc/xh_split.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_split.cpp,v
retrieving revision 1.1
diff -b -u -2 -r1.1 xh_split.cpp
--- src/xrc/xh_split.cpp	2003/01/26 23:18:09	1.1
+++ src/xrc/xh_split.cpp	2003/06/12 00:03:52
@@ -24,4 +24,6 @@
 #include "wx/log.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxSplitterWindowXmlHandler, wxXmlResourceHandler)
+
 wxSplitterWindowXmlHandler::wxSplitterWindowXmlHandler() : wxXmlResourceHandler()
 {
Index: src/xrc/xh_stbmp.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_stbmp.cpp,v
retrieving revision 1.5
diff -b -u -2 -r1.5 xh_stbmp.cpp
--- src/xrc/xh_stbmp.cpp	2002/08/20 22:28:15	1.5
+++ src/xrc/xh_stbmp.cpp	2003/06/12 00:03:52
@@ -23,4 +23,6 @@
 #include "wx/statbmp.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxStaticBitmapXmlHandler, wxXmlResourceHandler)
+
 wxStaticBitmapXmlHandler::wxStaticBitmapXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_stbox.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_stbox.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_stbox.cpp
--- src/xrc/xh_stbox.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_stbox.cpp	2003/06/12 00:03:52
@@ -23,4 +23,6 @@
 #include "wx/statbox.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxStaticBoxXmlHandler, wxXmlResourceHandler)
+
 wxStaticBoxXmlHandler::wxStaticBoxXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_stlin.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_stlin.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_stlin.cpp
--- src/xrc/xh_stlin.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_stlin.cpp	2003/06/12 00:03:52
@@ -25,4 +25,6 @@
 #if wxUSE_STATLINE
 
+IMPLEMENT_DYNAMIC_CLASS(wxStaticLineXmlHandler, wxXmlResourceHandler)
+
 wxStaticLineXmlHandler::wxStaticLineXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_sttxt.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_sttxt.cpp,v
retrieving revision 1.4
diff -b -u -2 -r1.4 xh_sttxt.cpp
--- src/xrc/xh_sttxt.cpp	2001/12/29 16:14:04	1.4
+++ src/xrc/xh_sttxt.cpp	2003/06/12 00:03:52
@@ -23,4 +23,6 @@
 #include "wx/stattext.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxStaticTextXmlHandler, wxXmlResourceHandler)
+
 wxStaticTextXmlHandler::wxStaticTextXmlHandler() 
 : wxXmlResourceHandler() 
Index: src/xrc/xh_text.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_text.cpp,v
retrieving revision 1.5
diff -b -u -2 -r1.5 xh_text.cpp
--- src/xrc/xh_text.cpp	2002/09/13 17:22:34	1.5
+++ src/xrc/xh_text.cpp	2003/06/12 00:03:52
@@ -23,4 +23,6 @@
 #include "wx/textctrl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxTextCtrlXmlHandler, wxXmlResourceHandler)
+
 wxTextCtrlXmlHandler::wxTextCtrlXmlHandler() : wxXmlResourceHandler() 
 {
Index: src/xrc/xh_toolb.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_toolb.cpp,v
retrieving revision 1.7
diff -b -u -2 -r1.7 xh_toolb.cpp
--- src/xrc/xh_toolb.cpp	2003/01/18 23:49:28	1.7
+++ src/xrc/xh_toolb.cpp	2003/06/12 00:03:52
@@ -26,4 +26,6 @@
 #if wxUSE_TOOLBAR
 
+IMPLEMENT_DYNAMIC_CLASS(wxToolBarXmlHandler, wxXmlResourceHandler)
+
 wxToolBarXmlHandler::wxToolBarXmlHandler() 
 : wxXmlResourceHandler(), m_isInside(FALSE), m_toolbar(NULL)
Index: src/xrc/xh_tree.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_tree.cpp,v
retrieving revision 1.6
diff -b -u -2 -r1.6 xh_tree.cpp
--- src/xrc/xh_tree.cpp	2002/09/15 18:06:55	1.6
+++ src/xrc/xh_tree.cpp	2003/06/12 00:03:52
@@ -23,4 +23,5 @@
 #include "wx/treectrl.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxTreeCtrlXmlHandler, wxXmlResourceHandler)
 
 wxTreeCtrlXmlHandler::wxTreeCtrlXmlHandler() 
Index: src/xrc/xh_unkwn.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_unkwn.cpp,v
retrieving revision 1.7
diff -b -u -2 -r1.7 xh_unkwn.cpp
--- src/xrc/xh_unkwn.cpp	2003/03/31 22:02:15	1.7
+++ src/xrc/xh_unkwn.cpp	2003/06/12 00:03:52
@@ -78,4 +78,6 @@
 
 
+IMPLEMENT_DYNAMIC_CLASS(wxUnknownWidgetXmlHandler, wxXmlResourceHandler)
+
 wxUnknownWidgetXmlHandler::wxUnknownWidgetXmlHandler()
 : wxXmlResourceHandler()
Index: src/xrc/xh_wizrd.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xh_wizrd.cpp,v
retrieving revision 1.1
diff -b -u -2 -r1.1 xh_wizrd.cpp
--- src/xrc/xh_wizrd.cpp	2003/04/22 20:25:36	1.1
+++ src/xrc/xh_wizrd.cpp	2003/06/12 00:03:52
@@ -26,4 +26,5 @@
 #include "wx/wizard.h"
 
+IMPLEMENT_DYNAMIC_CLASS(wxWizardXmlHandler, wxXmlResourceHandler)
 
 wxWizardXmlHandler::wxWizardXmlHandler() : wxXmlResourceHandler()
Index: src/xrc/xmlres.cpp
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/contrib/src/xrc/xmlres.cpp,v
retrieving revision 1.34
diff -b -u -2 -r1.34 xmlres.cpp
--- src/xrc/xmlres.cpp	2003/05/07 19:02:27	1.34
+++ src/xrc/xmlres.cpp	2003/06/12 00:03:52
@@ -144,4 +144,5 @@
 
 
+IMPLEMENT_ABSTRACT_CLASS(wxXmlResourceHandler, wxObject)
 
 void wxXmlResource::AddHandler(wxXmlResourceHandler *handler)

 	  	 
